R/twas.R

```diff
@@ -370,6 +371,7 @@ twas_pipeline <- function(twas_weights_data,
     twas_weights_data[[weight_db]][["molecular_id"]] <- weight_db
     twas_data_qced_result <- harmonize_twas(twas_weights_data[[weight_db]], ld_meta_file_path, gwas_meta_file)
     twas_data_qced <- twas_data_qced_result$twas_data_qced
+    if(is.null(twas_data_qced)) return(NULL)
```

Comment:
Same issue here. you need to comment with a warning that "No imputable gene-context pairs are left"; then return NULL